content/css/concepts/typography/terms/font-face/font-face.md

```diff
@@ -53,3 +57,4 @@ div {
   font-family: superFont;
 }
 ```
+**Note:**  Another thing to note while using any font is to check for browser support. If the format that we use is not supported, it will just roll back to the default system font. 
```

Comment:
```suggestion

> **Note:**  While using any font, check for browser support. If the format is not supported, it will use the default system font. 
```